dvc/output/base.py

```diff
@@ -297,8 +297,9 @@ def checkout(self, force=False, progress_callback=None, tag=None):
             progress_callback=progress_callback,
         )
 
-    def remove(self, ignore_remove=False):
-        self.remote.remove(self.path_info)
+    def remove(self, ignore_remove=False, ask=False):
```

Comment:
we can actually call it `force` here, don't we?

Reply:
okok :+1: